public/app/features/datasources/components/picker/DataSourceCard.tsx

```diff
@@ -98,7 +97,19 @@ function getStyles(theme: GrafanaTheme2, builtIn = false) {
       color: theme.colors.border.weak,
     }),
     selected: css({
-      backgroundColor: theme.colors.background.secondary,
+      background: theme.colors.action.selected,
+
+      '&::before': {
+        backgroundImage: theme.colors.gradients.brandVertical,
+        borderRadius: theme.shape.radius.default,
+        content: '" "',
+        display: 'block',
+        height: '100%',
+        position: 'absolute',
+        transform: 'translateX(-50%)',
+        width: theme.spacing(0.5),
+        left: 0,
```

Comment:
should this be 
```suggestion
        left: theme.spacing(0.25),
```
feels too close to the edge?